src/match.py

```diff
@@ -0,0 +1,15 @@
+class Match:
+    def __init__(self, stadium, attendance, date, host, guest, events, statistics):
```

Comment:
Set default value for some parameters. For example, **statistics** or **events** are empty until the match is played.
The same is true for the **attendance** parameter.